contact.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
+</head>
+<body>
+    <nav>
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="contact.html">Contact</a>
+          </li>
+        </ul>
+      </nav>
+
+     <h1>Amber Comfort</h1>
+
+      <h2>Reach Out</h2>
+
+      <img src="images/mailbox.jpg" alt="image of mailbox for email"/>
+
+      <h4>amber.comfort@wayne.edu</h4>
```

Comment:
I would wrap the email and phone number with an `<address>` tag. They are not headings.